srv/salt/ceph/ganesha/files/ganesha.conf.j2

```diff
@@ -8,6 +8,10 @@ EXPORT
 	Pseudo = /cephfs/;
 
 	Access_Type = RW;
+        
+        Protocols = 4;
+
+        Transports = TCP;
```

Comment:
Is there any scenario in which NFSv4 would use UDP? (Not sure if it has a "compatibility" mode for communicating with NFSv2/3 UDP clients.)